python/pythonlab/unittest_runner/unittest_runner/validation_protocol.py

```diff
@@ -0,0 +1,36 @@
+from neighborhood import World, NeighborhoodContextType
+from .stdout.stdout_tracker import StdoutTracker
+
+class ValidationProtocol(object):
+
+  def __init__(self):
+    self.stdout_tracker = StdoutTracker()
+
+  def get_stdout_logs(self, file_path = None) -> list[str]:
```

Comment:
I think this refactoring of `ValidationProtocol` structure is definitely an improvement.